TRENZ.Docs.API/Services/NavNodeOrderingService.cs

```diff
@@ -49,7 +53,7 @@ private async Task SetOrderByParent(NavNode node, Dictionary<string[], ISourceFi
             await SetChildrenOrderByOrderFile(node.LocationParts, node.Children.Values, orderFiles, cancellationToken);
         }
 
-        private async Task SetChildrenOrderByOrderFile(IEnumerable<string> pathParts,
+        private async Task SetChildrenOrderByOrderFile(IEnumerable<string> locationParts,
                                                        IEnumerable<NavNode> children,
```

Comment:
Maybe a per-parent log message here

    _logger.LogDebug("Reordering tree for {Parent}");

Reply:
I don't think that's necessary.